### ISSUE TITLE
Robot_Toolkit: Make CountourLoad IElement2D and implement required methods

### ISSUE THREAD

**Issue by IsakNaslundBh** [BHoM/Robot_Toolkit#186]
Make the ContourLoad into an IElement2D and implement the methods reuired for this to allow to make use of additional methods.

@pawelbaran can you when you have time please quickly collate the list of the required methods for the element types?

**Comment by pawelbaran** [comment]
Yep, see [#122](https://github.com/BuroHappoldEngineering/ModelLaundry_Toolkit/issues/122).

**Comment by IsakNaslundBh** [comment]
Migrating this to BHoM_Engine as objects have been moved over to main BHoM
